docs/pages/contributing/documentation/best-practices.mdx

```diff
@@ -0,0 +1,86 @@
+## Best practices and common mistakes
+
+This section highlights best practices and common mistakes to avoid.
+
+### Make every word count
+
+Concise sentences are easier to read, comprehend, and translate.
+
+- Use simple words with precise meanings.
+- Remove words that don't add substance.
+- Avoid using passive *to be verbs* like *been*, and *being*.
+- Avoid weak or vague verbs, such as *have*, *make*, and *do*.
+
+Whenever possible, choose words that have one clear meaning. When in doubt, choose the 
+simple word or phrase over a more formal or complex one.
+Avoid flowery language and unnecessary words whenever possible.
+For example:
+
+| Choose this | Instead of this |
+| --- | --- |
+| use | utilize, make use of |
+| remove | extract, take away, eliminate |
+| tell | inform, let know |
+| to | in order to, as a means to |
+| also | in addition |
+| connect | establish connectivity |
+
+Avoid adjectives and adverbs—words that describe how, when, or where. They're tempting, but 
+rarely add much value. Unless they're important to the meaning of a statement, leave them out.
+
+### Be consistent
+
+Use one term consistently to represent one concept. 
+For example, if you use tenant, cluster, and proxy interchangeably or ambiguously, you'll 
```

Comment:
Should we put these words in quotes here to be explicit that we're referring to them as words?